testing/refpolicy/APKBUILD

```diff
@@ -0,0 +1,58 @@
+# Maintainer: Tycho Andersen <tycho@docker.com>
+pkgname=refpolicy
+pkgver=20170204
+pkgrel=0
+pkgdesc="SELinux policy reference"
+url="https://github.com/TresysTechnology/refpolicy/wiki"
+arch="noarch"
+license="GPLv2"
+depends=""
+depends_dev=""
+makedepends="$depends_dev checkpolicy python gawk"
+install=""
+subpackages="$pkgname-doc"
+source="https://raw.githubusercontent.com/wiki/TresysTechnology/refpolicy/files/refpolicy-2.$pkgver.tar.bz2"
+builddir="$srcdir/refpolicy"
+
+# refpolicy config
+monolithic=n
+# This causes various distro-specific policies to be built. Since most SELinux
+# policies are written against redhat-based distros, we set that here.
+distro=redhat
+unknown_perms=deny
+
+# These are somewhat related to what is in the CentOS spec file, although they
+# are slightly differnet in what they install.
+makeCmds() {
+	make UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=1024 bare || return 1
```

Comment:
As above, it would probably be good to document what `MLS_CATS` and `MCS_CATS` are.

It would be *nice* to have a knob to allow rebuilding the package with UBAC support.